test/Jsr292/src/com/ibm/j9/jsr292/api/MethodHandleAPI_dropArgumentsToMatch.java

```diff
@@ -321,15 +325,16 @@ public static void test_dropArgumentsToMatch_normal_sublist_boolean() throws Thr
 	 */
 	@Test(groups = { "level.extended" })
 	public static void test_dropArgumentsToMatch_normal_sublist_byte() throws Throwable {
-		MethodHandle h1 = MethodHandles.lookup().findVirtual(Hash_Methods.class, "hash_byte", MethodType.methodType(String.class, byte.class, byte.class));
+		MethodHandle h1 = MethodHandles.lookup().findVirtual(Hash_Methods.class, "hash_byte", MethodType.methodType(String.class, byte[].class, byte.class));
 
 		/* Method with rtype = String, ptype = [boolean, char, int, long, byte, String] */
 		MethodType typeList = MethodType.methodType(String.class, boolean.class, char.class, int.class, long.class, byte.class, String.class);
 
 		h1 = MethodHandles.dropArgumentsToMatch(h1, 2, typeList.parameterList(), 4);
 
 		Hash_Methods hm = new Hash_Methods();
-		Assert.assertEquals(h1.invoke(hm, (byte)97, true, 'c', 1, 2, (byte)98, "d"), "ab", "Method handle did not return expected result");
+		byte[] input = new String("ab").getBytes();
```

Comment:
The original code was:
```
h1.invoke(hm, (byte)97, true, 'c', 1, 2, (byte)98, "d")
```

Does a minimal change like this pass on an ebcdic system?
```
h1.invoke(hm, (byte)'a', true, 'c', 1, 2, (byte)'b', "d")
```

The code here mixes the ascii number for the character and the actual character value.  Always converting the char into a byte in the code should be correct for any charset, I think.

Reply:
the dynamic casting of `char` to `byte` won't work directly since EBCDIC system value for readable letters (A-Z, a-z, 0-9) are in the range of 128 - 255, which is out of range for a `byte` (which is signed) to store and results in a negative value after casting (undefined characters when casted back to string).

Reply:
I'm hesitating on this PR as it changes the meaning of the test case.  The point of this test case is to validate the `dropArgumentsToMatch` using `byte`, not `byte[]`.

Can you modify the test case in a way to preserves using a `byte` as the signature types?  Feel free to drop the use of converting this to a String.

Reply:
The `byte` testing is preserved through the second part of the `byte[]` (`input[input.length-1]`) passed to the hash method, the goal of this test is to confirm the alignment when relocating different size parameters.

The reason for getting `byte[]` from `String` is to maintain the readability of the test result, as EBCDIC will translate `"ab"` to negative `byte` value while dynamic casting `(byte)'a'` produces the ASCII code of 'a'

the assert can still be done similar to
`hash_byte( (byte)'a', (byte)'b' ) == String(new byte[] { (byte)'a', (byte)'b' })`
which is less readable as the actual string build from the byte array is "/<?>" since 0x61 is undefined in EBCDIC